<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PWCI Auto Login https://pwcindonesia.com</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="elementClientPath">
        <property name="text">
         <string>ElementClient</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="elementClientLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="elementClientButton">
        <property name="text">
         <string>Cari</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="accountLabel">
        <property name="text">
         <string>Akun</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="charLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="charLabel">
        <property name="text">
         <string>Char</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="accountLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Sandi</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="addCharButton">
          <property name="text">
           <string>Tambah Akun</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearFormButton">
          <property name="text">
           <string>Hapus Akun</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTableView" name="charsTableView">
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="logCharButton">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteCharButton">
          <property name="text">
           <string>Hapus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="logAllCharsButton">
          <property name="text">
           <string>Login Semua Akun</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+S, Ctrl+S, Ctrl+S</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="unfreezeCheckbox">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;pre style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a name=&quot;tw-target-text-container&quot;/&gt;&lt;span style=&quot; font-family:'Courier New'; font-weight:700;&quot;&gt;M&lt;/span&gt;&lt;span style=&quot; font-family:'Courier New'; font-weight:700;&quot;&gt;encegah rendering berhenti saat jendela kehilangan fokus.&lt;/span&gt;&lt;/pre&gt;&lt;pre style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Courier New'; font-weight:700;&quot;&gt;Peringatan: Meningkatkan penggunaan CPU/GPU!&lt;/span&gt;&lt;/pre&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Unfreeze</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Bantuan</string>
    </property>
    <addaction name="actionBug"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBug">
   <property name="text">
    <string>Lapor Bug</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Tentang</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>elementClientLineEdit</tabstop>
  <tabstop>elementClientButton</tabstop>
  <tabstop>accountLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>charLineEdit</tabstop>
  <tabstop>addCharButton</tabstop>
  <tabstop>clearFormButton</tabstop>
  <tabstop>charsTableView</tabstop>
  <tabstop>logCharButton</tabstop>
  <tabstop>deleteCharButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
